owncloudApp/src/main/res/layout/manage_accounts_dialog.xml

```diff
@@ -1,68 +1,82 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There should be a good reason why using `RelativeLayout` 😨

Reply:
There is no reason for using `RelativeLayout. Replaced` by `ConstraintLayout`